src/pages/register-a-beacon/land-communications.tsx

```diff
@@ -0,0 +1,247 @@
+import { FormPageProps, handlePageRequest } from "../../lib/handlePageRequest";
+import { Communication } from "../../lib/registration/types";
+import { ofcomLicenseUrl, PageURLs } from "../../lib/urls";
+
+const definePageForm = ({
+  portableVhfRadio,
+  portableVhfRadioInput,
+  satelliteTelephone,
+  satelliteTelephoneInput,
+  mobileTelephone,
+  mobileTelephoneInput1,
+  mobileTelephoneInput2,
+  otherCommunication,
+  otherCommunicationInput,
+}: FormSubmission): FormManager => {
+  return new FormManager({
+    portableVhfRadio: new FieldManager(portableVhfRadio),
+    portableVhfRadioInput: new FieldManager(
+      portableVhfRadioInput,
+      [
+        Validators.required(
+          "We need your portable MMSI number if you have a portable VHF/DSC radio"
+        ),
+        Validators.mmsiNumber(
+          "Your portable MMSI number must be exactly nine digits long and only include numbers 0 to 9, with no letters or other characters"
+        ),
+      ],
+      [
+        {
+          dependsOn: "portableVhfRadio",
+          meetingCondition: (value) =>
+            value === Communication.PORTABLE_VHF_RADIO,
+        },
+      ]
+    ),
+    satelliteTelephone: new FieldManager(satelliteTelephone),
+    satelliteTelephoneInput: new FieldManager(
+      satelliteTelephoneInput,
+      [
+        Validators.required(
+          "We need your phone number if you have a satellite telephone"
+        ),
+        Validators.phoneNumber(
+          "Enter a satellite telephone number in the correct format"
+        ),
+      ],
+      [
+        {
+          dependsOn: "satelliteTelephone",
+          meetingCondition: (value) =>
+            value === Communication.SATELLITE_TELEPHONE,
+        },
+      ]
+    ),
+    mobileTelephone: new FieldManager(mobileTelephone),
+    mobileTelephoneInput1: new FieldManager(
+      mobileTelephoneInput1,
+      [
+        Validators.required(
+          "We need your telephone number if you have a mobile telephone"
+        ),
+        Validators.phoneNumber(
+          "Enter a mobile telephone number, like 07700 982736 or +447700912738"
+        ),
+      ],
+      [
+        {
+          dependsOn: "mobileTelephone",
+          meetingCondition: (value) => value === Communication.MOBILE_TELEPHONE,
+        },
+      ]
+    ),
+    mobileTelephoneInput2: new FieldManager(mobileTelephoneInput2),
+    otherCommunication: new FieldManager(otherCommunication),
+    otherCommunicationInput: new FieldManager(
+      otherCommunicationInput,
+      [
+        Validators.required("We need your other communication"),
+        Validators.maxLength(
+          "Other communication has too many characters",
+          250
+        ),
+      ],
+      [
+        {
+          dependsOn: "otherCommunication",
+          meetingCondition: (value) => value === Communication.OTHER,
+        },
+      ]
+    ),
+  });
+};
+
+const LandOtherCommunications: FunctionComponent<FormPageProps> = ({
```

Comment:
```suggestion
const LandCommunications: FunctionComponent<FormPageProps> = ({
```
Little update, same thing at the bottom where we're exporting